### PR TITLE
Print QR code of the private key

### PR DESCRIPTION
**Pull request by mrtn28**
Some services like [iris.to](https://iris.to/) allows login with a QR code.
This feature adds an option to display a QR of the private key when a valid key is found.

- `--qr` Print QR code of the private key

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rana"
-version = "0.3.6"
```
Comment (grunch) [inline Cargo.toml:3]: Please don't bump the version

File: src/main.rs [hunk 2]
```diff
@@ -251,3 +257,14 @@ fn get_leading_zero_bits(bytes: &[u8]) -> u8 {
     }
     res
 }
+
+fn print_qr(secret_key: SecretKey) {
+    let private_hex = secret_key.display_secret().to_string();
+    let code = QrCode::new(private_hex).unwrap();
```
Comment (grunch) [inline src/main.rs:263]: Can you please show the `nsec` instead hex private key? we are moving to work with `npub/nsec`.

**Review by grunch — CHANGES_REQUESTED** [review]
Great add, thanks for this contribution, please check change requests and update this branch from main

**Comment by mrtn28** [comment]
Sure, no problem

**Review by grunch — APPROVED** [review]
Thanks, merging it!

**Pull request merged**